packages/server/lib/browsers/webkit-automation.ts

```diff
@@ -322,12 +298,19 @@ export class WebKitAutomation {
    * Clear all cookies
    * @returns cookies cleared
    */
-  private async clearCookies (): Promise<CyCookie[]> {
-    const allCookies = await this.getCookies()
+  private async clearCookies (cookiesToClear: CyCookie[]): Promise<CyCookie[]> {
+    const allCookies = await this.context.cookies()
+    const persistCookies = allCookies.filter((cookie) => {
```

Comment:
any benefit to adding debug logs here to what cookies are cleared that fit the filter?

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/24471/commits/42ae0a13e96ae4ccfe73b0b8579cf9875fa68321